usaspending_api/api_docs/api_documentation/award_spending/recipient.md

```diff
@@ -0,0 +1,129 @@
+## [Award Spending Recipient](#usaspending-api-documentation)
```

Comment:
This route is not actively used in the frontend but has attachments to it.

FOR THIS ONE: remove from docs BUT NOT FROM API.

Reply:
removed from docs but api endpoint still active